@xen-orchestra/lite/src/libs/xen-api.ts

```diff
@@ -383,6 +384,15 @@ export default class XenApi {
           )
         );
       },
+      clone: (vmRefsToClone: VmRefsToClone) => {
+        const vmRefs = Object.keys(vmRefsToClone);
+
+        return Promise.all(
+          castArray(vmRefs).map((vmRef) =>
```

Comment:
`Object.keys()` return an array.
```suggestion
          vmRefs.map((vmRef) =>
```